opm/simulators/flow/FlowProblemBlackoil.hpp

```diff
@@ -435,6 +435,14 @@ class FlowProblemBlackoil : public FlowProblem<TypeTag>
             simulator.model().applyInitialSolution();
             FlowProblemType::writeOutput(true);
         }
+
+        if (!eclState.getIOConfig().initOnly()) {
+            if (!this->enableTuning_ && eclState.getSimulationConfig().anyTUNING()) {
+                OpmLog::warning("The deck has TUNING in the SCHEDULE section, but "
+                                "it is ignored due\nto the flag --enable-tuning=false. "
+                                "Set this flag to true to activate it.");
+            }
```

Comment:
I suggest adding a line: "Manually tuning the simulator with the TUNING keyword may increase run time. We recommend using the simulator's default tuning and not using the TUNING flag."